Learning_test_automation/Spotkanie6_Zadania/CurrencyConversion_vBeta/Decide/Currency.cs

```diff
@@ -1,27 +1,24 @@
 ﻿namespace CurrencySelection
 {
-    public class Currency//użyć "Czegoś" żeby nie tworzyć obiektu klasy Currency
+    public abstract class Currency
     {
-        public decimal BoughtCurrency { get; private set; }
-        public decimal SoldCurrency { get; private set; }
-        public decimal UserCurrency { get; private set; }
+        public decimal BoughtCurrency { get; set; }
+        public decimal SoldCurrency { get; set; }
+        public string NameCurrency { get; set; }
```

Comment:
Musi to byc typu string z przekazywaniem na sztywno tekstu w poszczegolnych konstruktorach?
Dodales swoj enum `CurrencyDefault` i lepiej byloby uzyc takiego typu tutaj i taki typ przekazywac jako argument.

Reply:
NameCurrency już tutaj nie ma xD muszę domergować zmiany xD

Reply:
udało się